tests/end-to-end/event/phpt-clean.phpt

```diff
@@ -24,8 +24,6 @@ Test Prepared (%sphpt-clean.phpt)
 Child Process Started
 Child Process Finished
 Test Passed (%sphpt-clean.phpt)
-Child Process Started
-Child Process Finished
```

Comment:
here we can see the new feature in action. a pre-existing test no longer needs a subprocess to cleanup